docs/source/algorithms/GetDetectorOffsets-v1.rst

```diff
@@ -42,6 +51,23 @@ single group on :ref:`algm-DiffractionFocussing`. The
 :ref:`algm-CreateCalFileByNames` algorithm can be used to
 alter the grouping in the cal file.
 
+Estimating the CC-spectrum fit parameters
+#########################################
+
+The starting value for the fit is that the linear background is zero.
+
+The peak center is found in two steps. First is the x-value of the heighest point in the whole cross-correlation spectrum.
+Then it is observed again by subtracting the background (which is zero) then finding the heighest point again within the neighborhood of the original data.
+
+The height is the background subtracted y-value of the heightest point as described earlier.
+
+The full-width-half-maximum is :math:`2 \sigma \sqrt{2 ln(2)}` where :math:`\sigma=10` if ``EstimateFWHM=False``.
+When ``True``, the value is estimated by integrating the background subtracted area under the window and dividing by the height.
+
+**Problems can arise** for fitting if the cross-correlation peak is broad or the background is high and/or with significant slope.
+Since a broad peak can be estimated as a very broad peak with a relatively high background, the peak can be easily misfit.
+Similarly, since the starting value for the background is zero, a heavily sloping background will interfere with determining a starting value for the peak position.
+
```

Comment:
```suggestion
Also note that if the input cross-correlation has been calculated over too large a d-spacing range it can contain "harmonics" in addition to the fundamental peak and this will obviously confuse the peak fitting algorithm.
```